tests/integration-tests/test_runner.py

```diff
@@ -61,6 +61,8 @@
     "cookbook_git_ref": None,
     "node_git_ref": None,
     "ami_owner": None,
+    "available_amis_oss_x86": [],
```

Comment:
Why an empty list rather than a None?
It seems to me that the more correct values are:
1. SUPPORTED_OSES: in fact you declare that if the option is omitted, the default behaviour will be to use all OSes.
2. None: to represent you're omitting the value

Reply:
Default to `[]` is for the simplicity that in all usage, we can assume list type, and do list operations without checking `None` type

Reply:
I see, but, correct me if I'm wrong, here you're setting the default value for an option, and the default value for this option is actually the full list of OSes, so why not simply setting it to `SUPPORTED_OSES `?